actions/slack-payload/src/libs/workflow-context/workflow-context.ts

```diff
@@ -54,15 +54,16 @@ const getBaseContext = async (
 ): Promise<Omit<WorkflowData, 'pullRequest'>> => {
   const jobsData = await getJobsData(token, workflowRunContext.workflow_run.id);
   const attachmentsData = await getAttachmentsData(token, workflowRunContext.workflow_run.id);
-  const environmentArtifactId = getEnvironmentAttachmentId(attachmentsData);
-  const environment = environmentArtifactId
-    ? await getEnvironment(token, environmentArtifactId)
-    : 'Unknown';
+  const workflowInfoArtifactId = getWorkflowInfoAttachmentId(attachmentsData);
+  const workflowInfoJson = workflowInfoArtifactId
+    ? await getWorkflowInfo(token, workflowInfoArtifactId)
+    : '{}';
+  const workflowInfo = JSON.parse(workflowInfoJson);
```

Comment:
See also my suggestion for getWorkflowInfo
```suggestion
  const workflowInfo = workflowInfoArtifactId
    ? await getWorkflowInfo(token, workflowInfoArtifactId)
    : addMissingWorkflowInfoProperties({});
```